pkg/controller/jobs/statefulset/statefulset_webhook.go

```diff
@@ -79,13 +80,17 @@ func (wh *Webhook) Default(ctx context.Context, obj runtime.Object) error {
 			ss.Spec.Template.Annotations = make(map[string]string, 1)
 		}
 		ss.Spec.Template.Annotations[pod.SuspendedByParentAnnotation] = FrameworkName
-		if ss.Spec.Template.Labels == nil {
-			ss.Spec.Template.Labels = make(map[string]string, 2)
-		}
 		queueName := jobframework.QueueNameForObject(ss.Object())
 		if queueName != "" {
+			if ss.Spec.Template.Labels == nil {
+				ss.Spec.Template.Labels = make(map[string]string, 2)
+			}
 			ss.Spec.Template.Labels[constants.QueueLabel] = queueName
-			ss.Spec.Template.Labels[pod.GroupNameLabel] = GetWorkloadName(ss.Name)
+			groupName, err := GetWorkloadName(obj.(*appsv1.StatefulSet))
```

Comment:
This change does not seem releated to changing queue-name. Please revert if not necessary.

Reply:
Agree. Created separate PR #3951.